tests/QueryBuilderTest.php

```diff
@@ -676,27 +676,32 @@ public function testUpdateSubdocument()
     public function testDates()
     {
         DB::table('users')->insert([
-            ['name' => 'John Doe', 'birthday' => new UTCDateTime(Date::parse('1980-01-01 00:00:00'))],
-            ['name' => 'Robert Roe', 'birthday' => new UTCDateTime(Date::parse('1982-01-01 00:00:00'))],
-            ['name' => 'Mark Moe', 'birthday' => new UTCDateTime(Date::parse('1983-01-01 00:00:00.1'))],
-            ['name' => 'Frank White', 'birthday' => new UTCDateTime(Date::parse('1960-01-01 12:12:12.1'))],
+            ['name' => 'John Doe', 'birthday' => Date::parse('1980-01-01 00:00:00')],
+            ['name' => 'Robert Roe', 'birthday' => Date::parse('1982-01-01 00:00:00')],
+            ['name' => 'Mark Moe', 'birthday' => Date::parse('1983-01-01 00:00:00.1')],
+            ['name' => 'Frank White', 'birthday' => Date::parse('1975-01-01 12:12:12.1')],
```

Comment:
Changed the date to avoid the issue reported in [PHPC-2429](https://jira.mongodb.org/browse/PHPC-2429)